policy/cnspec_policy.proto

```diff
@@ -147,6 +147,41 @@ message Bundle {
   repeated Framework frameworks = 8;
   repeated FrameworkMap framework_maps = 9;
   PolicyDocs docs = 5;
+  // Migrations for the current bundle, describes changes to its contents
+  // from older versions or other collections.
+  // Experimental as of v11.x
+  repeated MigrationGroup migration_groups = 11;
+}
+
+// MigrationGroup contains a set of migrations for a policy collection.
+// The policy should generally contain the UID and affected version.
+message MigrationGroup {
+  Policy policy = 1; // only informational fields will be used (version)
+  repeated Migration migrations = 2;
+}
+
+// Migration is an action performed on a matching policy.
+// This is generally performed to track changes in UIDs from old policies
+// to new policies.
+message Migration {
+  enum Action {
+    UNSPECIFIED = 0;
+    CREATE = 1;
+    REMOVE = 2;
+    MODIFY = 3;
+  }
+
+  MigrationMatch match = 1;
```

Comment:
whats the match for?

Reply:
These are the conditions on which a check is matched for a migration. Imagine throwing all checks at a migration, when this condition is true (i.e. when it matches) then the appropriate action is run.